draft-porfiri-dhc-dhcpv4-over-dhcpv6-ra.md

```diff
@@ -249,40 +285,42 @@ it might be enough to only use 4o6RA, as shown in {{fig_4o6RAserver}}.
           | L2 Network  |
  +--------+-+         +-+------+----------+
  |   DHCP   |         |  4o6   | DHCP 4o6 |
- |  Client  +---------+  RA    +  Server  |
- |  on CPE  |         |        |          |
+ |  Client  +---------+  Relay +  Server  |
+ |  on CPE  |         |  Agent |          |
  +--------+-+         +-+------+----------+
           |             |
            '-----------'
 
 ~~~
-{: #fig_4o6RAserver title="Topology path preserved 4o6 RA in DHCP server" artwork-align="center"}
+{: #fig_4o6RAserver title="Topology path preserved 4o6 Relay Agent in DHCP server" artwork-align="center"}
 
 # Deployment Considerations
 
-As the client is not aware of the 4o6RA, the network deployment needs to ensure that
-all DHCPv4 broad- and unicast messages from the client are routed over the 4o6RA.
-This can e.g. be achieved by placing the 4o6RA in a central position that can observe all traffic
+As clients are not aware of the presence of 4o6RA, the network deployment needs to ensure that
+all DHCPv4 broadcast and unicast messages from clients are steered to a 4o6RA.
+This can, e.g., be achieved by placing the 4o6RA in a central position that can observe all traffic
 from the clients or use of address translation with the 4o6RA address for unicast
 messages.
 
 # Security Considerations {#seccons}
 
-This documents applies 4o6 DHCP in a scenario where legacy IPv4 clients are
+This documents specifies the applicability of 4o6 DHCP in a scenario where legacy IPv4 clients are
 connected to 4o6 DHCP Relay Agent that performs the en- and decapsulation. This document
 does not change anything else in the 4o6 DHCP specification and therefore the
 security consideration of {{RFC7341}} still apply.
 
-The mechanism described differs from {{RFC7341}} as the DHCP client
-actually sends and receveis DHCP messages, whereas in {{RFC7341}} it
+This mechanism differs from {{RFC7341}} as the DHCP client
+sends and receives DHCPv4 messages, whereas in {{RFC7341}} it
 only sends DHCPv6 messages. This makes it possible that
-DHCP messages could reach a DHCP server without using the 4o6RA.
-While this can cause errornous states in both the client and DHCP server
-and potentially even lead to misconfigrations that impact reachability,
-this is not seen as a security concern rather than a deloyment error.
-
-More generally, the legacy IPv4 client is not aware of this mechanism, however, even
-when 4o6 DHCP is used, the client does not have any control about the
+DHCPv4 messages could reach a DHCPv4 server without using the 4o6RA.
+While this can cause erroneous state in both clients and servers
+and potentially even lead to misconfigurations that impact reachability,
+this is not seen as a security concern rather than a deployment error
+and even though it may be used for attacks from within the network,
+it's not new and it's not being introduced because of this specification.
```

Comment:
```suggestion
this is not seen as a security concern rather than a deployment error.
Further, even though it may be used for attacks from within the network,
this is not new and it is not introduced because of this specification.
```